cp3pt0-deployment/setup_tenant.sh

```diff
@@ -157,6 +157,7 @@ function pre_req() {
         success "oc command logged in as ${user}"
     fi
 
+    create_ns_list
```

Comment:
I believe we could try to move following functions out of `pre_req` function
- `create_ns_list`
- `check_cert_manager`
- `check_licensing`

IMHO, `pre_req` function could be used for validate scripts parameters only.

And we have a a new function to validate necessary functional prerequisite, after we `setup_tepology`

- `pre_req`
- `setup_topology`
-  `check_singleton_service`
    - `check_cert_manager`
    - `check_licensing`